nixos-config/ren/disko.nix

```diff
@@ -1,41 +1,39 @@
-_: let
```

Comment:
Well, this *is* certainly nicer, but untested, and should probably be in a separate PR.